### DISCUSSION TITLE
Check if tab changed

### DISCUSSION THREAD

**Discussion by VLTNOgithub** [TomSchimansky/CustomTkinter#984] [Q&A] [answered]
Hi, this is my first time posting here.
Is it possible to check if a tab changed?

Thanks

**Comment by abhaymakes** [comment, marked as answer]
Yes, the tab widget accepts a command parameter, you could attach a function to it which gets called when a tab is switched.

**Reply by VLTNOgithub** [reply]
Edit: Got it. Added it in the CTKTabView function. 
Before: `tabview = customtkinter.CTkTabview(self, height=200)`
After: `tabview = customtkinter.CTkTabview(self, height=200, command=functionWhenTabChanged)`
